lib/stages/run_tests.sh

```diff
@@ -11,7 +11,19 @@ _stage_run_tests() {
     project_location=$(get_project_location)
 
     local test_selection=""
-    local phpunit=${DRUPAL_TESTING_DRUPAL_INSTALLATION_DIRECTORY}/${composer_bin_dir}/phpunit
+    local phpunit=""
+
+    if ${DRUPAL_TESTING_PARALLEL_TESTING}; then
```

Comment:
I think if you do a set env in GH actions for this var, it evaluates to a string and then "false" is also true here